pkgs/development/compilers/urweb/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
     sha256 = "1cl0x0sy7w1lazszc8q06q3wx0x0rczxh27vimrsw54s6s9y096s";
   };
 
-  buildInputs = [ openssl mlton libmysqlclient postgresql sqlite icu ];
+  buildInputs = [ openssl mlton libmysqlclient postgresql sqlite icu.dev ];
```

Comment:
Is this necessary? I thought `buildInputs` already pick up all relevant outputs.

Reply:
this change won't be necessary though using `.dev` on the other line is https://github.com/NixOS/nixpkgs/pull/79219/files#diff-cb0ea61e5234748808950ee5ba2112f1R26. I can revert this line back if you want.

Reply:
Yes, please do revert that if it's unnecessary. LGTM otherwise, thanks!

Reply:
done